<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QListWidget" name="chatLists">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="chatBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>481</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageText">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>310</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-next"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
